chandra_models/xija/dea/dea_spec.json

```diff
@@ -253,7 +253,17 @@
             799
         ]
     },
-    "limits": {},
+    "limits": {
+        "1deamzt": {
+            "odb.caution.high": 39.5,
+            "odb.caution.high.matlab_identifier": "yellow_limit",
+            "odb.warning.high": 42.5,
+            "odb.warning.high.matlab_identifier": "red_limit",
+            "planning.warning.high": 37.5,
+            "planning.warning.high.matlab_identifier": "reference_limit",
```

Comment:
The ACIS models are oddballs in labeling the planning limit as `reference_limit`.